procurement_mto_analytic/models/purchase_order_line.py

```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    def _merge_in_existing_line(
+            self, product_id, product_qty, product_uom, location_id, name,
+            origin, values):
+        # Do not merge into a line with a different analytic account
+
```

Comment:
Remove blank lines inside a method. It's a waste of space and doesn't add anything valuable.